extensions/Webhooks/template/en/default/pages/webhooks.html.tmpl

```diff
@@ -0,0 +1,190 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is  |  "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% PROCESS global/header.html.tmpl
+   title = "Webhooks Documentation"
+   style = "#bugzilla-body li {
+              margin: 5px
+            }
+            h4 {
+              margin-bottom: 0px
+            }
+            .heading {
+              font-weight: bold
+            }
+            #main-inner{
+              margin-left: 20%;
+              margin-right: 20%;
+              margin-bottom: 5%
+            }
+            "
+%]
+
+<h1>[% terms.Bugzilla %] Webhooks</h1>
+
+<p>A webhook is a custom callback defined by events. Is triggered when those events happen, and a POST
+request is sent to a defined URL.</p>
+
+<p>In the case of [% terms.Bugzilla %], a webhook can be triggered by a change to, or creation of a [% terms.bug %]. The
+parameters of the [% terms.bug %] are exposed to the webhook handler which makes a callback (over HTTP) to another
+web application.</p>
+
+<p>Examples of [% terms.Bugzilla %] webhooks could include:</p>
+<ul>
+  <li>Updating a copy of the [% terms.Bugzilla %] [% terms.bug %] in another system such as Jira</li>
+  <li>Sending a message to a chat server such as Matrix or Slack</li>
+</ul>
+
+<h2>Creating a webhook</h2>
+<p>To create a webhook you just need to follow the next steps:</p>
+<ol>
+  <li>Access to your [% terms.Bugzilla %] Account.</li>
+  <li>Go to your Preferences Panel > Webhooks.</li>
+  <li>Fill out all of the parameters:</li>
+    <h4>Name</h4>
+    <p>A name for you to recognize the webhook. (e.g. “My Jira webhook”)</p>
```

Comment:
`A name for the webhook, which should be descriptive (e.g. `Jira Webhook for New & Updated Bugs in Core::Graphics`